### PR TITLE
Fixcontinue 

### PR DESCRIPTION
**Pull request by jrharris620**
Solved the public abstract ShouldContinue with a public override for the _fixcontinue_ branch of the **GitKata**.

### CODE REVIEW

File: GitKata/Program.cs [hunk 1]
```diff
@@ -16,27 +13,82 @@ static void Main(string[] args)
...
-                ques =  kata.AnswerQuestionAndGetNext(ans);
+                ques = kata.AnswerQuestionAndGetNext(ans);
 
             }
 
             Console.WriteLine("Congratulations, you've defeated me.  Press enter to quit");
             Console.ReadLine();
+
         }
+
```
Comment (codytwinton) [inline GitKata/Program.cs]: Let's remove this unnecessary whitespace. More readable.

Comment (jrharris620) [reply GitKata/Program.cs]: Cleaned up the whitespace.
